validation/utils/compare_parquet_csv.py

```diff
@@ -23,22 +25,31 @@ def is_sorting_type(dtype):
             pd.api.types.is_integer_dtype(dtype))
 
 
-def compare(parquet_path, csv_path, sort, **kwargs):
+def compare(parquet_path, csv_path, sort, lower_case_cols, rasie_error, **kwargs):
     parquet_df = pd.read_parquet(parquet_path)
     dtypes = parquet_df.dtypes.apply(lambda x: x.name).to_dict()
     csv_df = pd.read_csv(csv_path, dtype=dtypes)
 
+    if lower_case_cols:
+        parquet_df.columns = parquet_df.columns.str.lower()
+        csv_df.columns = csv_df.columns.str.lower()
+
     if sort:
         sort_column = [column for column, dtype in dtypes.items() if is_sorting_type(dtype)]
         parquet_df = parquet_df.sort_values(by=sort_column)
         csv_df = csv_df.sort_values(by=sort_column)
 
-    pd.testing.assert_frame_equal(parquet_df, csv_df, **kwargs)
+    try:
+        pd.testing.assert_frame_equal(parquet_df, csv_df, **kwargs)
+    except AssertionError as e:
+        print(e)
```

Comment:
Maybe
if raise_error:
  raise
else:
   print(e)
to not have the error printed twice